README.md

```diff
@@ -169,8 +169,6 @@ If you like this project you can support any time.
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPIONAL] Kindly remove the Server Express.js and PostgreSQL Database from the Tech Stack section. We're not using them in the project. 🚮 


![image](https://user-images.githubusercontent.com/79058364/235209953-0a6ec92c-77cb-48d9-ada1-1d7fa7efdd76.png)